ampersand-select-view.js

```diff
@@ -173,8 +182,20 @@ SelectView.prototype.remove = function () {
     this.el.removeEventListener('change', this.onChange, false);
 };
 
+SelectView.prototype.reset = function() {
+    if(this.startingValue) {
+        this.setValue(this.startingValue, true);
+    } else {
+        this.clear();
+    }
+};
+
 SelectView.prototype.clear = function() {
-    this.setValue('', true);
+    if(this.unselectedText) {
+        this.setValue(undefined, true);
```

Comment:
@cdaringe I saw you comment on #34 about `value` should always being a `string`. Do you think I should change this back to `setValue('', true)` for consistencies sake?